test/support/mockConfig.js

```diff
@@ -1,26 +1,143 @@
 const { mock } = require('pactum');
 const { like } = require('pactum-matchers');
-const sheResponse = require('../fixtures/Bundle-DDCC-TX-SHE-response-example-1.json');
+
+const operationOutcomeError = require('../fixtures/OperationOutcome-example.json');
+const rhcResponse = require('../fixtures/Response-ProvideDDCCDocument.json');
 
 const config = {
   setup() {
-    mock.addInteraction({
-      id: 'submitHealthEvent',
-      strict: false,
-      request: {
-        method: 'POST',
-        path: '/submitHealthEvent',
-        body: {
-          resourceType: like('Bundle'),
-          id: like('DDCC-TX-SHE-bundle-example-1')
-        }
-      },
-      response: {
-        status: 201,
-        body: sheResponse
-      }
-    });
-  }
-}
+    mock.addInteraction([
+      {
+        id: 'storeHealthCertificate',
+        strict: false,
+        request: {
+          method: 'POST',
+          path: '/Bundle',
+          body: {
+            resourceType: 'Bundle',
+            id: like('Example-English'),
+            type: 'document',
+          },
+        },
+        response: {
+          status: 201,
+          headers: {
+            location: '/DDCCDocument/TESTID',
+          },
+        },
+      },
+      {
+        id: 'storeInvalidHealthCertificate-400',
+        strict: false,
+        request: {
+          method: 'POST',
+          path: '/Bundle',
+          body: {
+            resourceType: 'Bundle',
+            id: 'Bundle-DDCCDocument-400',
+          },
+        },
+        response: {
+          status: 400,
+          body: operationOutcomeError,
+        },
+      },
+      {
+        id: 'storeInvalidHealthCertificate-404-1',
+        strict: false,
+        request: {
+          method: 'POST',
+          path: '/INVALIDENDPOINT',
+        },
+        response: {
+          status: 404,
+          body: operationOutcomeError,
```

Comment:
At some point we might want to make an `/OperationOutcome` subdirectory in `test/fixtures` that includes Operation Outcomes with different issues codes (http://hl7.org/fhir/R4/operationoutcome-definitions.html#OperationOutcome.issue.code) that correspond to the different error codes that could be returned by the response, but it's probably not integral to this PR.

The current `OperationOutcome-example.json` uses the [issue-type code](http://hl7.org/fhir/R4/valueset-issue-type.html) `"required"`, but I don't think this code would be returned for something like an invalid endpoint, for example.

Reply:
I added additional json files for the "invalid" code and "not found" code - I think this will cover all the error cases for now

Reply:
Nice! Looks good - I'm sure we may learn more about more specific details included within the Operation Outcome once we begin testing with the actual server reference implementation.